src/Infrastructure.Styles.Analyzer/RMSYN1103ExtensionMethodsTestedAsNormalInvocationCodeFixProvider.cs

```diff
@@ -0,0 +1,130 @@
+        {
+            var diagnostic = context.Diagnostics.First();
+            var diagnosticSpan = diagnostic.Location.SourceSpan;
+
+            var syntaxRoot = await context.Document.GetSyntaxRootAsync(cancellationToken);
+            if (syntaxRoot == null)
+                return context.Document;
+
+            var oldNode = syntaxRoot.FindNode(diagnosticSpan).FirstAncestorOrSelf<InvocationExpressionSyntax>()!;
+
+            var semanticModel = await context.Document.GetSemanticModelAsync(cancellationToken);
+            if (semanticModel == null)
+                return context.Document;
+
+            var symbolInfo = semanticModel.GetSymbolInfo(oldNode, cancellationToken);
+            if (symbolInfo.Symbol is not IMethodSymbol methodSymbol)
+                return context.Document;
+
+            var containingType = methodSymbol.ContainingType;
+            if (containingType == null)
+                return context.Document;
+
+            var expression = CreateInvocationExpressionNamespaceTree(oldNode, containingType);
+            if (expression == null)
+                return context.Document;
+
+            var invocation = (MemberAccessExpressionSyntax)oldNode.Expression;
+
+            var nodeWithNewArguments = NodeWithNewFirstArgument(oldNode.WithExpression(expression), SyntaxFactory.Argument(invocation.Expression));
+
+            var newSyntaxRoot = syntaxRoot.ReplaceNode(oldNode, nodeWithNewArguments);
+            
+            return context.Document.WithSyntaxRoot(newSyntaxRoot);
+        }
+
+        private static MemberAccessExpressionSyntax? CreateInvocationExpressionNamespaceTree (InvocationExpressionSyntax oldNode, INamedTypeSymbol containingType)
+        {
+            var invocationChildExpression = (MemberAccessExpressionSyntax)oldNode.Expression;
+
+            var stack = new Stack<SimpleNameSyntax>();
+            stack.Push(invocationChildExpression!.Name);
+            stack.Push(SyntaxFactory.IdentifierName(containingType.Name));
+
+            var containingNamespace = containingType.ContainingNamespace;
+            while (containingNamespace != null)
+            {
+                if (!containingNamespace.IsGlobalNamespace && (containingNamespace.Name != string.Empty))
+                    stack.Push(SyntaxFactory.IdentifierName(containingNamespace.Name));
+                containingNamespace = containingNamespace.ContainingNamespace;
+            }
+
+            if (stack.Count < 2)
+                return null;
+
+            var newExpression = SyntaxFactory.MemberAccessExpression(SyntaxKind.SimpleMemberAccessExpression,
+                stack.Pop(), stack.Pop()).WithAdditionalAnnotations(Simplifier.Annotation);
+
+            while (stack.Any())
+                newExpression =
+                    SyntaxFactory.MemberAccessExpression(SyntaxKind.SimpleMemberAccessExpression, newExpression,
+                        stack.Pop());
```

Comment:
```suggestion
            var newExpression = SyntaxFactory.MemberAccessExpression(
                SyntaxKind.SimpleMemberAccessExpression,
                stack.Pop(),
                stack.Pop())
              .WithAdditionalAnnotations(Simplifier.Annotation);

            while (stack.Any())
            {
                newExpression = SyntaxFactory.MemberAccessExpression(
                    SyntaxKind.SimpleMemberAccessExpression, newExpression,
                    stack.Pop());
            }
```

Multiline statements always with braces, and if we have a linebreak in the parameters break before the first and after each parameter except the last.